src/shared_modules/content_manager/src/actionOrchestrator.hpp

```diff
@@ -80,6 +82,11 @@ class ActionOrchestrator final
                     std::stoi(m_spBaseContext->spRocksDB->getLastKeyValue().second.ToString());
             }
 
+            if (offset == 0)
```

Comment:
What happens when `offset` is greater than `0`?

Reply:
The check is done in `OnDemandManager.cpp`, nd in this case if the `offset` is greater than `0` just keep the last value, `std::stoi(m_spBaseContext->spRocksDB->getLastKeyValue().second.ToString());`

Reply:
Thanks!